Pipfile

```diff
@@ -5,6 +5,17 @@ verify_ssl = true
 
 [dev-packages]
 cosmpy = { editable = true, extras = ["dev", "test"], path = "." }
+darglint = "==1.8.0"
+bandit = "==1.7.0"
+black = "==21.6b0"
+isort = "==5.9.3"
+flake8 = "==3.9.2"
+mypy = "==0.910"
+pyflakes = "==2.3.1"
+pytest = "==7.0.1"
+tox = "==3.24.1"
+vulture = "==2.3"
+
```

Comment:
At some point we decided to add all dev dependencies in setup.py and exclude it from Pipfile. You can see dangling is listed in setup.py already.

Reply:
fixed